.github/workflows/dotnet-core-linux.yml

```diff
@@ -12,15 +12,24 @@ jobs:
 
     strategy:
       matrix:
-        version: [netcoreapp3.1]
+        version: [netcoreapp2.1,netcoreapp3.1]
 
     steps:
     - uses: actions/checkout@v2
 
-    - name: Setup .NET Core
+    - name: Setup .NET Core 2.1.807
```

Comment:
simplify name by removing the .807 part.

Reply:
i will do for all places.